bedrock/base/templates/macros.html

```diff
@@ -189,7 +189,7 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
 {%- endmacro %}
 
 {# Docs: https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#linking-to-monitorfirefoxcom #}
-{% macro monitor_button(entrypoint, form_type='button', utm_campaign=None, utm_content=None, utm_term=None, button_text=None, button_class='mzp-c-button mzp-t-product', button_id='fxa-monitor-submit') -%}
+{% macro monitor_button(entrypoint, form_type='button', utm_campaign=None, utm_content=None, utm_term=None, button_text=None, button_class='mzp-c-button mzp-t-product', button_id='fxa-monitor-submit', cta_type=None, cta_position=None) -%}
```

Comment:
It would be great if we can try and keep changes to these macros documented here: https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#linking-to-monitor-firefox-com

Reply:
Oh good catch, I'll add it to the todo list!